docs/versioned_docs/version-v3/providers/strava.md

```diff
@@ -13,7 +13,7 @@ The **Strava Provider** comes with a set of default options:
 
 - [Strava Provider options](https://github.com/nextauthjs/next-auth/blob/main/src/providers/strava.js)
 
-You can override any of the options to suit your own use case.
+You can override any of the options to suit your own use case. Ensure the `redirect_uri` configuration fits your needs accordingly. 
```

Comment:
@michaelangrivera my bad, but this was actually done in the wrong file! This is the v3 (unmaintained) documentation. Could you apply this change here in a follow-up PR: https://github.com/nextauthjs/next-auth/blob/main/docs/docs/providers/strava.md

Reply:
WIll do!

Reply:
@balazsorban44  FYI https://github.com/nextauthjs/next-auth/pull/5299